### DISCUSSION TITLE
没有秃头基因的燊 - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#22] [Announcements]
# 没有秃头基因的燊 - 没有秃头基因的燊的博客

喜爱折腾的人 - 没有秃头基因的燊 - 没有秃头基因的燊的博客

https://yzs020220.github.io/about/

**Comment by Mashiro726** [comment]
想问下师傅，关于live2d_widget的使用，我在git clone到source里面后，去找主题目录layout文件夹中的head文件在哪，我目录里面有关的只有一个head.pug

**Reply by yzs020220** [reply]
具体要看你的主题是什么，一般来讲就是在head这个文件里去加代码即可
